nebula-cache-manager/core/classes/CacheManager.cls

```diff
@@ -367,6 +376,15 @@ public without sharing class CacheManager {
       return this.keyToValue.get(key);
     }
 
+    public Object get(String key, System.Type cacheBuilderClass) {
+      if (this.contains(key) == false) {
+        Cache.CacheBuilder cacheBuilder = (Cache.CacheBuilder) cacheBuilderClass.newInstance();
+        Object value = cacheBuilder.doLoad(key);
+        this.put(key, value);
```

Comment:
I love this block of code ❤️ This is a great way to leverage cacheBuilder in the transaction cache